media/formats/mp4/box_definitions.cc

```diff
@@ -1580,15 +1580,21 @@ IamfSpecificBox::IamfSpecificBox(const IamfSpecificBox& other) = default;
 IamfSpecificBox::~IamfSpecificBox() = default;
 
 FourCC IamfSpecificBox::BoxType() const {
-  return FOURCC_IAMF;
+  return FOURCC_IACB;
 }
 
 bool IamfSpecificBox::Parse(BoxReader* reader) {
-  const int obu_bitstream_size = reader->box_size() - reader->pos();
+  uint8_t configuration_version;
+  RCHECK(reader->Read1(&configuration_version));
+  RCHECK(configuration_version == 0x01);
+
+  uint32_t config_obus_size;
+  RCHECK(ReadLeb128Value(reader, &config_obus_size));
+
   const uint8_t* buf = reader->buffer() + reader->pos();
-  ia_descriptors.assign(buf, buf + obu_bitstream_size);
+  ia_descriptors.assign(buf, buf + config_obus_size);
```

Comment:
This is different, do we have base::span in Cobalt?

  base::span<const uint8_t> buffer =
      reader->buffer().subspan(reader->pos(), config_obus_size);
  ia_descriptors.assign(buffer.begin(), buffer.end());

Reply:
Not sure why I made that change, I'll check if we do and upload a patch if it works

Reply:
We do have `base::span`, but `reader->buffer()` was only changed to a `span` recently: https://source.chromium.org/chromium/chromium/src/+/8dada69952e8e8752c4d76e52c8b5b426e6745fd. To avoid updating all other uses of `buffer()` in this file I've added a guard.